osf_tests/test_node.py

```diff
@@ -3888,6 +3904,18 @@ def test_update_share_correctly_for_projects(self, requests, node, user, request
             graph = kwargs['json']['data']['attributes']['data']['@graph']
             assert graph[1]['is_deleted'] == case['is_deleted']
 
+    @mock.patch('website.project.tasks.settings.SHARE_URL', 'https://share.osf.io')
+    @mock.patch('website.project.tasks.settings.SHARE_API_TOKEN', 'Token')
+    @mock.patch('website.search.search.update_collected_metadata')
+    @mock.patch('website.project.tasks.requests')
```

Comment:
I'm reasonably certain that neither this nor the SHARE settings need to be mocked.

Reply:
Yeah this all gets moved in https://github.com/CenterForOpenScience/osf.io/pull/9348/files so I'll deal with the conflicts then because all these tests got DRYed up.